### PR TITLE
Home

### PR DESCRIPTION
**Pull request by GarvG**
Please Review my PR made my Home Page for E commerce

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (zoyatanweer) [inline .vscode/settings.json:3]: you can delete this file or put in .gitignore

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jordan| HOME</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@1,300&family=Poppins:wght@400;600;700&family=Raleway:wght@300&family=Roboto:ital,wght@1,300&display=swap"
+        rel="stylesheet">
```
Comment (zoyatanweer) [inline Index.html]: you can import this google file into your common or main css file, so this way you won't need to add it again and again into all your html file.

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                        <li><a href=""></a>LOGIN</li>
+                    </ul>
+                </nav>
+
+
+            </div>
+            <div class="row">
+                <div class="col-2">
+                    <h1>GET YOURSLEF<br> A NEW BEAST PAIRS!</h1>
+                    <p>I want to do for rebounds<br> what Michael Jordan did for dunks. - Dennis Rodman</p>
```
Comment (zoyatanweer) [inline Index.html:39]: change the color of this para. It is not easily readable in the homepage.

**Review by zoyatanweer — APPROVED** [review]
do the required changes and then you are good to merge.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    margin-right: 1.5rem;
+    font-size:1.2rem;
+}
+a{
+    text-decoration: none;
+    color:#555;
+}
+p
+{
+    color:#555
```
Comment (archit7448) [inline styles.css]: I think you will use same color again and again why not use var

File: .vscode/settings.json (hunk 1 above)
Comment (GarvG) [reply .vscode/settings.json:3]: okay

File: Index.html (hunk 2 above)
Comment (GarvG) [reply Index.html]: yeah made another css file main.css imported there

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jordan| HOME</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!-- <link
+        href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@1,300&family=Poppins:wght@400;600;700&family=Raleway:wght@300&family=Roboto:ital,wght@1,300&display=swap"
+        rel="stylesheet"> -->
```
Comment (zoef-yt) [inline Index.html]: Remove comment code.

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+}
+nav ul li{
+    display: inline-block;
+    margin-right: 1.5rem;
+    font-size:1.2rem;
+}
+a{
+    text-decoration: none;
+    color:#555;
```
Comment (zoef-yt) [inline styles.css]: Use var

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+}
+p
+{
+    color:white;
```
Comment (zoef-yt) [inline styles.css]: Use var

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+.col-2 h1
+{
+    font-size: 3.1rem;
+    line-height: 3.7rem;
+    margin: 2rem 0px;
+}
+.btn{
+    display: inline-block;
+    background: black;
+    color:#fff;
```
Comment (zoef-yt) [inline styles.css]: Use var

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+.col-2
+{
+    flex-basis: 50%;
+    min-width: 18rem;
+}
+.col-2 img{
+    max-width: 100%;
+    padding: 3.5rem;
+}
+
```
Comment (zoef-yt) [inline styles.css:74]: Remove spacing

File: styles.css [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+    background: black;
+    color:#fff;
+    padding: 0.5rem 2.2rem;
+    margin: 2.2rem 0;
+    border-radius: 2.2rem;
+
+}
+.header
+{
+    background: radial-gradient(white,black);
```
Comment (zoef-yt) [inline styles.css]: Use var

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: styles.css (hunk 10 above)
Comment (GarvG) [reply styles.css]: done

File: styles.css (hunk 9 above)
Comment (GarvG) [reply styles.css:74]: removed

File: styles.css (hunk 8 above)
Comment (GarvG) [reply styles.css]: used var

File: styles.css (hunk 7 above)
Comment (GarvG) [reply styles.css]: done

File: styles.css (hunk 6 above)
Comment (GarvG) [reply styles.css]: Used Var

File: Index.html (hunk 5 above)
Comment (GarvG) [reply Index.html]: Removed

**Review by zoef-yt — APPROVED** [review]
Good to merge.

File: styles.css (hunk 4 above)
Comment (GarvG) [reply styles.css]: done using var

**Pull request merged**